Comp2023/src/main/java/frc/robot/subsystems/Elbow.java

```diff
@@ -197,14 +189,14 @@ private void initSmartDashboard( )
     // Initialize Variables
     if (m_elbowDebug)
```

Comment:
I'm not sure there's a reason to hide these behind "m_elbowDebug" anymore. This initialization code is only called when we switch into Teleop or Auto the first time. (Actually, since these are just putting constants to the dashboard, I question whether they are even useful to do that anymore. They don't change ever.)